.github/workflows/wheels.yml

```diff
@@ -13,7 +11,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: ['3.7', '3.8', '3.9', '3.11', 'pypy3.7', 'pypy3.8', 'pypy3.9', '3.10']
```

Comment:
This should be irrelevant, since the wheel is built in any case inside a `manylinux` container.

Not sure how to implement it with `sed`...
I know something algorithmic is more convenient (even with some changes to the base, `sed` would keep working), but if it is too complex I'd rather go for a patch, or just commit the final result manually edited.

Reply:
About the irrelevancy I'm not so sure anymore, it seems `setup-python` may also have an effect there: <https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-setup-python-with-a-self-hosted-runner>. We should figure out the effect and do it properly as you said.

Reply:
A *self-hosted runner* is not a container, but a different machine you provide on your own to run the workflow, instead of using those of GitHub (i.e. Azure...).

Instead, `maturin-action` is running a container in a GitHub's virtual machine, that is relevant for the host system (we need a Linux machine to run a Linux container, otherwise we would need a Linux virtual machine inside the virtual machine), but it is not relevant Python-wise, since the `manylinux` container comes pre-built with its versions of Python.

This is true only for Linux, while for Windows and MacOS the `setup-python` is a crucial step.